.travis.yml

```diff
@@ -11,6 +11,7 @@ script:
   - cargo fmt --all -- --check
   - cd webxr; cargo build --features=glwindow
   - cd webxr; cargo build --features=ipc,glwindow
+  - cd webxr; cargo build --features=ipc,headless
```

Comment:
Can just make this `--features=glwindow,headless` and `--features=ipc,glwindow,headless`